docs/gangplank/api-cosa.md

```diff
@@ -829,18 +1028,20 @@ type Amis struct {
 
 ```go
 type Artifact struct {
-    Path               string `json:"path"`
-    Sha256             string `json:"sha256"`
-    SizeInBytes        int    `json:"size,omitempty"`
-    UncompressedSha256 string `json:"uncompressed-sha256,omitempty"`
-    UncompressedSize   int    `json:"uncompressed-size,omitempty"`
+    Path               string  `json:"path"`
+    Sha256             string  `json:"sha256"`
+    SizeInBytes        float64 `json:"size,omitempty"`
```

Comment:
why the need for floating point??

Reply:
Per the JSON spec, an `int` is a 32-bit int. And since some of our files are larger than 2.4gb, we have to render into a float64 to get enough bits. https://github.com/coreos/coreos-assembler/pull/2080

Reply:
Thanks for the educational material.